Dockerfile

```diff
@@ -34,14 +34,15 @@ RUN <<EOT
     mkdir -p /copy/status.d
 
     executables=( \
-        /usr/bin/omnect-cli \
-        /usr/sbin/fdisk \
         /usr/bin/dd \
-        /usr/bin/sync \
-        /usr/bin/e2mkdir \
         /usr/bin/e2cp \
+        /usr/bin/e2mkdir \
         /usr/bin/fallocate \
         /usr/bin/mcopy \
+        /usr/bin/omnect-cli \
+        /usr/bin/ssh-keygen \
+        /usr/bin/sync \
+        /usr/sbin/fdisk \
```

Comment:
bmaptool is missing?

Reply:
yes I removed since it doesn't work anyway. it is still in `apt-get` because omnect-cli.deb has a dependency to it.